easybuild/easyconfigs/m/MoviePy/MoviePy-1.0.1-foss-2019a-Python-3.7.2.eb

```diff
@@ -0,0 +1,41 @@
+# This easyconfig was created by Simon Branford of the BEAR Software team at the University of Birmingham.
+easyblock = 'PythonBundle'
+
+name = 'MoviePy'
+version = '1.0.1'
+versionsuffix = '-Python-%(pyver)s'
+
+homepage = "https://zulko.github.io/moviepy/"
+description = """MoviePy (full documentation) is a Python library for video editing: cutting, concatenations,
+ title insertions, video compositing (a.k.a. non-linear editing), video processing, and creation of custom effects."""
+
+toolchain = {'name': 'foss', 'version': '2019a'}
+
+dependencies = [
+    ('Python', '3.7.2'),
+    ('Pillow', '6.0.0'),
+    ('SciPy-bundle', '2019.03'),
+    ('tqdm', '4.32.1'),
+]
+
+use_pip = True
+
+exts_default_options = {'source_urls': [PYPI_SOURCE]}
+
+exts_list = [
+    ('imageio', '2.6.1', {
+        'checksums': ['f44eb231b9df485874f2ffd22dfd0c3c711e7de076516b9374edea5c65bc67ae'],
+    }),
+    ('imageio-ffmpeg', '0.3.0', {
```

Comment:
I'm a bit surprised that FFmpeg isn't a dependency for this package?

Reply:
Oops. On testing I get
```
RuntimeError: No ffmpeg exe could be found. Install ffmpeg on your system, or set the IMAGEIO_FFMPEG_EXE environment variable.
```

It seems that this does not get checked during the install - only at runtime. The user we installed this for is using it in conjunction with PyTorch and that has a FFmpeg dependency.

I'll get it fixed.

Reply:
With FFmpeg added as a dep:
```
$ python
Python 3.7.2 (default, Aug 28 2019, 11:29:43) 
[GCC 8.2.0] on linux
Type "help", "copyright", "credits" or "license" for more information.
>>> import imageio_ffmpeg
>>> imageio_ffmpeg.get_ffmpeg_exe()
'ffmpeg'
```